pkg/rule/rule.go

```diff
@@ -59,21 +59,21 @@ const (
 	PostExecRule Type = "postExecRule"
 )
 
-// pod is a simple type to encapsulate a pod's uid and namespace
-type pod struct {
-	uid       string
-	namespace string
+// Pod is a simple type to encapsulate a Pod's uid and namespace
+type Pod struct {
+	UID       string `json:"uid"`
+	Namespace string `json:"namespace"`
 }
 
 type podErrorResponse struct {
-	pod v1.Pod
+	Pod v1.Pod
 	err error
 }
 
-// commandTask tracks pods where commands for a taskID might still be running
-type commandTask struct {
-	taskID string
-	pods   []pod
+// CommandTask tracks pods where commands for a taskID might still be running
+type CommandTask struct {
+	TaskID string `json:"task_id"`
```

Comment:
taskID for the json tag to keep in line with k8s tags

Reply:
done